lib/_emerge/Scheduler.py

```diff
@@ -1953,26 +1953,21 @@ def _schedule_tasks_imp(self):
         @return: True if state changed, False otherwise.
         """
 
-        state_change = 0
-
         while True:
             if not self._keep_scheduling():
-                return bool(state_change)
+                return False
```

Comment:
If it added a package in the previous iteration of the loop, then that means the state changed, and it's supposed to return True so that the calling _schedule_tasks loop will continue to reconcile the state until there are no changes.

Reply:
I see, that is a good reason to keep it. I've dropped the according commit.

Reply:
I feel bad that I missed that. Modified the commit to simply convert state_change to boolean (which I like better, but it is probably subjective).